modules/ROOT/pages/tips-and-tricks.adoc

```diff
@@ -163,13 +163,17 @@ or setup shell `alias`es as needed to make them available to the CLI like so:
 
 == Enabling TPM2 for LUKS 
 
-It is possible to use the TPM2 device in your machine to automatically decrypt your LUKS-encrypted device. 
+It is possible to use the TPM2 device in your machine to automatically decrypt your LUKS-encrypted device. It is important to note a few things regarding this setup: first, it is trading convenience for security. Anyone who has access to both the disk and the TPM2 device can unlock your data. Second, it is crucial to keep a LUKS passphrase in addition to the TPM2 device that you can use to unlock the LUKS device; if you update BIOS, or the kernel shim, or make other substantial changes to your device, the TPM will refuse to sign your session and you will fall back to manually entering a LUKS passphrase. This tip assumes you have a single LUKS device created during the Anaconda installer; if you have more than one LUKS device, you will need to correctly identify the device you want to enroll. 
 
-First, enable the needed initramfs and kernel features: 
+To set up TPM2 unlocking, first, find the LUKS device you want to enroll. This is probably in /etc/cryptsetup; you can also use `cryptsetup status /dev/mappper/luks*` to identify the device.
+
+Next, enable the required initramfs and kernel features. Note that the initramfs command below will overwrite any other initramfs changes you have made: 
 
   $ sudo rpm-ostree kargs --append=rd.luks.options=tpm2-device=auto
   $ sudo rpm-ostree initramfs --enable --arg=-a --arg=systemd-pcrphase
 
 Then, enroll the device: 
 
-  $ sudo systemd-cryptenroll --tpm2-device=auto --tpm2-pcrs=0+7 /your/boot/device
+  $ sudo systemd-cryptenroll --tpm2-device=auto --tpm2-pcrs=0+7 /dev/mapper/luks-your-device
```

Comment:
`/dev/mapper/luks-your-device` here is confusing, because this argument should be the *underlying device* that your luks container lives in. e.g for me it's `/dev/nvme0n1p3`

Reply:
corrected; the 'cryptsetup status' command will show the correct device to enroll